src/azure-cli/azure/cli/command_modules/role/commands.py

```diff
@@ -99,6 +99,18 @@ def load_command_table(self, _):
         g.custom_command('credential list', 'list_application_credentials')
         g.custom_command('credential delete', 'delete_application_credential')
 
+    # Register federated-credential command group under both app and sp.
+    # We use federated-credential instead of federated-identity-credential or fic, in order to align with
+    # Azure Portal.
+    # It seems sp doesn't work with federatedIdentityCredentials yet.
+    # for item in ['app', 'sp']:
+    for item in ['app']:
+        with self.command_group(f'ad {item} federated-credential',
+                                client_factory=get_graph_client, resource_type=PROFILE_TYPE,
+                                exception_handler=graph_err_handler, is_experimental=True) as g:
+            for command in ['list', 'create', 'show', 'update', 'delete']:
+                g.custom_command(command, f'{item}_federated_credential_{command}')
```

Comment:
This actually inspires me of a new naming convention. 

Instead of use `create_application` for `az ad application create`:

https://github.com/Azure/azure-cli/blob/8f74ef4dfcc56722332ef7c6c498c1b885b9f63c/src/azure-cli/azure/cli/command_modules/role/commands.py#L85

the custom function can be named `app_create` to match the command name and the binding can be done in a uniform way.

Also, it becomes much easier to find the function in the function list:

![image](https://user-images.githubusercontent.com/4003950/173492561-ee8428ad-5f60-45a6-ba8e-887d0326c5d0.png)